### PR TITLE
Talk model

### PR DESCRIPTION
**Pull request by mimmon**
Talk model. Added also lightning talk choice, just in case. If unwanted, please create an issue, it will be removed.

### CODE REVIEW

File: konfera/models/talk.py [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
 
 class Talk(models.Model):
-    pass
+    title = models.CharField(max_length=256)
+    abstract = models.TextField()
+    talktype = models.CharField(max_length=32, choices=TALK_TYPE_CHOICES)
+    status = models.CharField(max_length=32, choices=TALK_STATUS_CHOICES)
+    duration = models.IntegerField(choices=DURATION_CHOICES)
+    primary_speaker = models.ForeignKey(Speaker)
+    secondary_speaker = models.ForeignKey(Speaker)
```
Comment (ZoltanOnody) [inline konfera/models/talk.py]: `secondary_speaker` should be optional.

File: konfera/models/talk.py [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+    event = models.ForeignKey(Event, blank=True)
```
Comment (ricco386) [inline konfera/models/talk.py]: All talks should reference some event, so please remove: blank=True

**Comment by ricco386** [comment]
@mimmon there was already MR by @TomasHo who added talk model. However it didnt had the idea to have a lightning talk as a choice, which I like, so if you could update from upstream and amend the model. Thanks.

**Comment by mimmon** [comment]
closing pull request, sent a new one

**Pull request closed without merging**